types/mempool/sender_nonce_test.go

```diff
@@ -192,3 +193,54 @@ func (s *MempoolTestSuite) TestTxNotFoundOnSender() {
 	err = mp.Remove(tx)
 	require.Equal(t, mempool.ErrTxNotFound, err)
 }
+
+func (s *MempoolTestSuite) TestUnorderedTx() {
+	t := s.T()
+
+	ctx := sdk.NewContext(nil, false, log.NewNopLogger())
```

Comment:
We should be able to just use `context.Background()` for this test

Reply:
We need an sdk.Context here, because below we call ctx.WithPriority when calling mp.Insert